src/static/riot/search.tag

```diff
@@ -765,10 +773,13 @@
                 overflow hidden
                 text-overflow ellipsis
                 max-width 90vw
+                display block !important
+                @media screen and (min-width 1921px)
```

Comment:
Where does 1921 come from, is this a common #?

Reply:
I was using 1921px to represent 'larger than 1920px'. I changed it to 2560px to comply with wider industry standards as shown in this write-up. https://css-tricks.com/optimizing-large-scale-displays/